### PR TITLE
ci: migrate to hercules

### PR DESCRIPTION
**Pull request by ghost**
https://discourse.nixos.org/t/hercules-ci-builder-for-aarch64-x86-64-available-in-nix-community/16028

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -1,15 +0,0 @@
...
-  tests:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - uses: cachix/install-nix-action@v14
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
-        extra_nix_config: |
-          experimental-features = nix-command flakes
-    - run: ./scripts/ci/test.sh
```
Comment (ghost) [inline .github/workflows/pr.yml:15]: TODO: we'll need to move this invocation, and potential flakes opt-in, to `ci.nix`.

**Comment by ghost** [comment]
I am stuck on how to invoke the scripts/run-all.nu from within ci.nix. Any changes welcome!

File: ci.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        {
+          buildInputs = [ pkgs.nushell ];
+        } ''
+        ls -l $out
+        nu scripts/run-all.nu > $out
+      '';
+    }
+  )
+{
+  x86_64-linux = { };
```
Comment (Mic92) [inline ci.nix:17]: btw. you can also enable `aarch64-linux`.

**Comment by ghost** [comment]
I'm still running into obscure errors so will abandon this effort for now.

**Pull request closed without merging**